python_base/Dockerfile

```diff
@@ -42,8 +42,11 @@ RUN yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.n
         python-virtualenv \
         wget && \
     yum clean all
-RUN npm install -g \
-        node-sass \
+RUN npm cache clean -f && \
+    npm install -g n && \
+    n 4.4.6 && \
```

Comment:
Hmmm, there's only 6.7.0 on yum repos, though it makes no sense that this is the issue as it was not failing before no?